packages/turbo/ios/RNVisitableView.swift

```diff
@@ -116,7 +106,7 @@ extension RNVisitableView: SessionDelegate {
     // Handle a visit proposal
     if (session.webView.url == proposal.url) {
```

Comment:
`webView.url` just?

Reply:
Actually, we get the `webView` from the passed-in `session` argument. While in theory, they point to the same, I guess it's cleaner to use the dependency from the argument as now.